deploy/docker/cp-api-srv/folder-templates/ROLE_USER/DefaultStorage/template.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@@",
+    "metadata": {
+        "type": {
+            "value": "defaultUserFolder"
+        },
+        "DataStorage": {
+            "value": "${CP_PREF_STORAGE_SCHEMA}://cloud-pipeline-user-@@-default-storage",
+            "type": "bucket"
+        }
+    },
+    "parentId" : "${CP_PARENT_FOLDER_ID_DEFAULT_USER_STORAGE}",
+    "permissions": [
```

Comment:
please remove `permissions` from the template, we need to handle it manually

Reply:
Done